code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -412,8 +412,24 @@
 
 /obj/item/weapon/robot_module/combat/New()
 	src.modules += new /obj/item/device/flash/cyborg(src)
+	src.modules += new /obj/item/weapon/restraints/handcuffs/cable/zipties/cyborg(src)
 	src.modules += new /obj/item/borg/sight/thermal(src)
-	src.modules += new /obj/item/weapon/gun/energy/laser/cyborg(src)
+	src.modules += new /obj/item/weapon/gun/energy/gun/cyborg(src)
+	src.modules += new /obj/item/weapon/gun/energy/plasmacutter(src)
+	src.modules += new /obj/item/borg/combat/shield(src)
+	src.modules += new /obj/item/borg/combat/mobility(src)
+	src.modules += new /obj/item/weapon/wrench(src) //Is a combat android really going to be stopped by a chair?

```

Comment:
cyborgs can infact type `resist` to get out of a chair
infact that's safer than wrenching it, wrenching it may trap you 5ever